package.json

```diff
@@ -63,6 +63,7 @@
     "@angular/platform-server": "~13.1.1",
     "@angular/router": "~13.1.1",
     "@nguniversal/express-engine": "^13.0.1",
+    "bootstrap": "^5.1.3",
```

Comment:
This module is totally decoupled from Design System UI Libraries such as Bootstrap. Could you remove it's reference, please?

Reply:
done!